docs/notebooks/point_clouds.ipynb

```diff
@@ -13,30 +13,28 @@
   },
```

Comment:
<p>I'd add a citation right after <code>... using the so-called entropic map ...</code> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>